src/routes/PrivateRoute.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import {Redirect, Route} from 'react-router-dom';
+import { useSelector } from 'react-redux';
+import { isLoaded, isEmpty } from 'react-redux-firebase';
+import {ApplicationState} from '../store/ApplicationState';
+import LoadingScreen from "../components/LoadingScreen/LoadingScreen";
+
+function PrivateRoute(props: any) {
+  const auth = useSelector((state: ApplicationState) => state.firebase.auth);
+  if (isLoaded(auth)) {
+     if (isEmpty(auth)) {
+       return <Redirect to='/new'/>;
```

Comment:
Maybe it would be nice to redirect the user to a dedicated `/login` site and redirect him back to his initial request on successful login, instead of redirecting each user to `/new`. What do you think?

The `/join/:id` route could be ommitted with this change.

Reply:
I totally agree with you. Until now the PrivateRoute is only being used for accessing a board, so it would make perfectly sense to redirect to the 'join' site instead. I will change it as soon as the join site is implemented 👍